ui/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "libretime-ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit --testMatch '**/*.spec.js'",
+    "lint": "vue-cli-service lint",
+    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'",
```

Comment:
Can I put this into a `"i18n"` section instead of putting it into this cli call?